web/themes/joinup/prototype/sass/components/_share-box.sass

```diff
@@ -1,60 +1,90 @@
 // Share box
 //
 
-// Elements
-//
-.share-box__item-icon
-  +icon-size(50px)
-  color: $color-blue-dark
-  display: none
+// Component
+// --------------------------------------------------
 
+// Elements
+// --------------------------------------------------
 .share-box__row
-  padding: 10px 50px 10px 10px
-  margin-bottom: 0 !important
   position: relative
-  min-height: 70px
-  &:hover
-    cursor: pointer
-  &.already-shared
-    padding-right: 0
-    background: $color-grey-lighter
-    .share-box__item-icon
-      display: table-cell
-      vertical-align: middle
-  .form-type-checkbox
-    .form__label
-      display: none
+  min-height: 50px
+  margin-bottom: 20px
+  &:last-child
+    border-bottom: none
+    min-height: auto
+    margin-bottom: 0
 
-.share-box__item-image
-  padding-right: 10px
-  img
-    max-width: 50px
-    max-height: 50px
+.share-box__item
+  text-decoration: none
 
-.share-box__item-image,
-.share-box__item-label,
-.share-box__item-social-icon
-  display: table-cell
-  vertical-align: middle
+.share-box__label
+  margin-left: 14px
+  color: $color-base
+  font-size: $font-size-base
+  font-weight: $font-weight
 
-.share-box__item-label
-  width: 100%
+.share-box__image img
+  max-width: 50px
+  max-height: 50px
 
-.share-box__item-social-icon
-  padding-right: 10px
-  +icon-size(50px)
+.share-box__social-icon
   color: $color-blue-dark
+  +icon-size(50px)
 
 .share-box__container
   max-width: $screen-sm
-  fieldset
-    margin-bottom: 0
   +breakpoint(tablet)
     min-width: $screen-sm
 
+.share-box__title
+  margin-top: 0
+  color: $color-base
+
+.share-box__social
+  margin-bottom: 20px
+  +breakpoint(sm)
+    display: inline-block
```

Comment:
Not a stopper, though it's probably unnecessary to use `inline-block` + width when you have the flexbox.

Reply:
Can you please take a look if you still can see additional scroll?